lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -1203,6 +1203,41 @@ def test_errorbar3d():
     ax.legend()
 
 
+@image_comparison(['stem3d.png'], style='mpl20')
+def test_stem3d():
+    theta = np.linspace(0, 2*np.pi)
+    x = np.cos(theta - np.pi/2)
+    y = np.sin(theta - np.pi/2)
+    z = theta
+
+    fig, axs = plt.subplots(1, 3, figsize=(8, 3),
+                            constrained_layout=True,
+                            subplot_kw={'projection': '3d'})
+    for ax, zdir in zip(axs, ['x', 'y', 'z']):
+        ax.stem(x, y, z, orientation=zdir)
+        ax.set_title(f'orientation={zdir}')
+
+
+@image_comparison(baseline_images=['stem3d_options.png'], style='mpl20',
+                  tol=0.006)
+def test_stem3d_options():
+    x = np.linspace(-np.pi/2, np.pi/2, 20)
+    y = np.ones_like(x)
+    z = np.cos(x)
+
+    fig, axs = plt.subplots(1, 3, figsize=(8, 3),
+                            constrained_layout=True,
+                            subplot_kw={'projection': '3d'})
+    for ax, zdir in zip(axs, ['x', 'y', 'z']):
+        markerline, stemlines, baseline = ax.stem(
+            x, y, z,
+            linefmt='C4-.', markerfmt='C1D', basefmt='C2',
+            orientation=zdir)
+        ax.set_title(f'orientation={zdir}')
+        markerline.set(markerfacecolor='none', markeredgewidth=2)
+        baseline.set_linewidth(3)
```

Comment:
A single baseline image test should be enough?  e.g. using defaults on orientation="x" and customized formats on other orientations.

Reply:
I squashed them into one image.